src/testing.ts

```diff
@@ -59,6 +59,10 @@ const expectedResults = new Map<string, ResultType>([
   ['example_contracts/idManglingTest8', 'Success'],
   ['example_contracts/idManglingTest9', 'Success'],
   ['example_contracts/if-flattening', 'Success'],
+  ['example_contracts/imports/importEnum', 'CairoCompileFailed'],
```

Comment:
Could you add comments explaining why the tests that don't succeed don't succeed